pkg/macvlan/macvlan.go

```diff
@@ -446,18 +470,22 @@ func macvlanCmdAdd(args *skel.CmdArgs) error {
 		}
 
 		if err := netlink.RouteAdd(&newDefaultRoute); err != nil {
-			logging.Errorf("failed to add new default route : %v", err)
-			return fmt.Errorf("failed to add new default route : %v", err)
+			// Check if we already have route installed
+			if !os.IsExist(err) {
+				logging.Errorf("failed to add new default route, gw %v : %v", gw, err)
```

Comment:
I see this has no relation to your patch, but this looks strange: this function should not return an error since it's never percolated up and logged anywhere else. Logging this here and returning an error seems strange and gives the impression  that we will have duplicated log lines. Anyways, this might deserve a clean-up patch later on.